pyrogram/client/methods/messages/send_audio.py

```diff
@@ -74,6 +75,11 @@ def send_audio(self,
             title (``str``, *optional*):
                 Track name.
 
+            thumb (``str``, *optional*):
+                Audio thumbnail.
```

Comment:
For such method parameters I usually take docstrings from the bot api docs (when they exist), with some small changes to adapt the texts or fix typos.